site/en/docs/crux/history-api/index.md

```diff
@@ -0,0 +1,617 @@
+The CrUX History API allows for the querying of a historty user experience metrics for a specific URI like "Get the history of time series metrics for the `https://example.com` origin."
+
+The History API follows the same structure as the daily [CrUX API](../api/) except values are given in an array, and keys are labelled with plural names (for example, `histogramTimeseries` instead of `histogram`, or `p75s` instead of `p75`).
+
+## CrUX API Key
+
+Like the daily API, using the CrUX History API requires a Google Cloud API key. The same key can be used for the daily and history API.
+
+You can create one in the [Credentials](https://console.developers.google.com/apis/credentials) page and provision it for `Chrome UX Report API` usage.
+
+After you have an API key, your application can append the query parameter `key=[YOUR_API_KEY]` to all request URLs. See the [Example queries](#example-queries) below.
+
+The API key is safe for embedding in URLs; it doesn't need any encoding.
+
+## Data model
+
+This section details the structure of data in requests and responses.
+
+### Record
+
+A discrete piece of information about a page, or site. A record can have data that is specific for an identifier and for a specific combination of dimensions. A record can contain data for one or more metrics.
+
+### Identifiers
+
+Identifiers specify what records should be looked up. In CrUX these identifiers are webpages and websites.
+
+### Origin
+
+When the identifier is an origin all data present for all pages in that origin are aggregated together. For example, say the `http://www.example.com` origin had pages as laid out by this sitemap:
+
+```text
+http://www.example.com
+http://www.example.com/foo.html
+http://www.example.com/bar.html
+```
+
+This would mean that when querying the Chrome UX Report with the origin set to `http://www.example.com`, data for `http://www.example.com/`, `http://www.example.com/foo.html`, and `http://www.example.com/bar.html` would be returned, aggregated together, because those are all pages under that origin.
+
+### URLs
+
+When the identifier is a URL, only data for that specific URL will be returned. Looking again to the `http://www.example.com` origin sitemap:
+
+```text
+http://www.example.com
+http://www.example.com/foo.html
+http://www.example.com/bar.html
+```
+
+If the identifier is set to URL with the value of `http://www.example.com/foo.html`, only data for that page will be returned.
+
+### Dimensions
+
+Dimensions identify a specific group of data that a record is being aggregated against. For example, a form factor of `PHONE` indicates that the record contains information about loads that took place on a mobile device.
+
+The CrUX History API is only available aggregated by form factor dimension. This is a general class of device split into `PHONE`, `TABLET`, and `DESKTOP`.
+
+{% Aside %}
+Unlike the daily API, the CrUX History API does not have data aggregated against Effective Connection Type (ECT).
+{% endAside %}
```

Comment:
These two paragraphs are saying the same thing. Can they be combined?

Reply:
Yes it's kind of implied in first paragraph as _only_ form factor is available, but I felt it important to explicitly highlight that ECT was not available as it is a difference from the daily API. You don't think this is necessary?